deploy/dev-ngsa-asb-eastus/loderunner/dev/loderunner/loderunner-api.yaml

```diff
@@ -30,6 +30,8 @@ spec:
           args:
           - --url-prefix
           - /api
+          - --cosmos-auth-type
```

Comment:
This can be removed. Let AutoGitOps update the deployment yaml files when it runs in the loderunner repository workflows on merge

Reply:
I have fixed this.